swift-service/src/main/java/com/facebook/swift/service/ThriftServiceProcessor.java

```diff
@@ -149,6 +149,7 @@ public void onSuccess(Boolean result)
                         @Override
                         public void onFailure(Throwable t)
                         {
+                            LOG.error("Failed to process method [" + method.getName() + "] of service [" + method.getServiceName() + "]", t);
```

Comment:
Won't this log all exceptions, including ones the method is allowed to throw (e.g. the ones defined in the IDL file)? We should probably log only those that are not being serialized and sent back to the client - those that are likely to terminate the connection